### PR TITLE
add support for reading motion tracking data from Xsens mvnx files

### PR DESCRIPTION
**Pull request by helenacockx**
These commits introduces a function to read mvnx-files from xsens motion capture systems. The commits are based on mvnx files that are released under version 4.

### CODE REVIEW

File: fileio/private/xsens_mvnx.m [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+  
+  % only select data that is asked for
+  dat = dat(chanindx, begsample:endsample);
+  
+  % return the data
+  varargout = {dat};
+  
+elseif needevt
+  %% parse the events
+  ft_warning('reading of events is not yet implemented');
```
Comment (robertoostenveld) [inline fileio/private/xsens_mvnx.m:321]: hier zouden misschien de stap-events en zo kunnen komen. Maar dat kan later nog wel.

Comment (helenacockx) [reply fileio/private/xsens_mvnx.m:321]: Ja, zeker! Maar eerst even nagaan of dit zou werken zo.

**Comment by robertoostenveld** [comment]
Could you add a `LICENSE` file or so (if available) to the external/xsens directory? And a `VERSION` file in case you know to which version of the XSens matlab code this corresponds? This will help in updating it in the future. 

See for example: 
```
mac036-lan> ls fieldtrip/external/*/LICENSE
bayesFactor/LICENSE	ezc3d/LICENSE		videomeg/LICENSE
biosig/LICENSE		mars/LICENSE		xdf/LICENSE
dss/LICENSE		netcdf/LICENSE
eeprobe/LICENSE		openmeeg/LICENSE

mac036-lan> ls fieldtrip/external/*/VERSION
biosig/VERSION		eeprobe/VERSION		videomeg/VERSION
ctf/VERSION		gifti/VERSION		wavefront/VERSION
dss/VERSION		vgrid/VERSION		yokogawa/VERSION
```

File: fileio/private/xsens_mvnx.m (hunk 1 above)
Comment (helenacockx) [reply fileio/private/xsens_mvnx.m:321]: Ik wilde trouwens een voorbeeld bijvoegen, maar het bestand is te groot...

Comment (robertoostenveld) [reply fileio/private/xsens_mvnx.m:321]: ziet er goed uit hoor!

**Comment by robertoostenveld** [comment]
Could you share an example mvnx file (not as attachement here, but through another channel) and write a simple `fieldtrip/test/test_ 1374.m` file that simply calls `ft_read_header` and `ft_read_data` on that file? The goal is to have something that (in the future) would give an error in case someone else makes changes to the format. See https://en.wikipedia.org/wiki/Regression_testing

**Comment by robertoostenveld** [comment]
I have merged your contribution into the master branch, and copied two test files to our DCCN network drive. It will automatically be tested this enening, and if there are no errors in any of the test scripts, it will be released.

See http://www.fieldtriptoolbox.org/development/dashboard/ for details on the testing and release mechanism.

See http://www.fieldtriptoolbox.org/getting_started/xsens/ for the documentation and add additional stuff there if you like.

**Pull request merged**